it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/policies/.content.xml

```diff
@@ -12,7 +12,7 @@
                       jcr:primaryType="nt:unstructured"
                       jcr:title="formcontainer-policy"
                       sling:resourceType="wcm/core/components/policy/policy"
-                      components="[/apps/forms-components-examples/components/form/textinput,/apps/forms-components-examples/components/form/datepicker,/apps/forms-components-examples/components/form/panelcontainer]">
+                      components="[/apps/forms-components-examples/components/form/textinput,/apps/forms-components-examples/components/form/datepicker,/apps/forms-components-examples/components/form/panelcontainer,/apps/forms-components-examples/components/form/fileinput,/libs/core/fd/components/form/fileinput/v1/fileinput,/libs/core/fd/components/form/textinput/v1/textinput]">
```

Comment:
it should be /apps path

Reply:
do you mean remove this /libs/core/fd/components/form/fileinput/v1/fileinput and add only /apps/forms-components-examples/components/form/fileinput

Reply:
yes.